nginx-config.yml.sample

```diff
@@ -1,45 +1,45 @@
-integration_name: com.newrelic.nginx
+integrations:
+- name: nri-nginx
+  env:
+    METRICS: "true"
+    # If you're using ngx_http_api_module be certain to use the full path up to and including the version number
+    STATUS_URL: http://127.0.0.1/status
+    # Name of Nginx status module OHI is to query against. discover | ngx_http_stub_status_module | ngx_http_status_module | ngx_http_api_module
+    STATUS_MODULE: discover
 
-instances:
-    - name: nginx-server-metrics
-      command: metrics
-      arguments:
-          # If you're using ngx_http_api_module be certain to use the full path up to and including the version number
-          status_url: http://127.0.0.1/status
-    
-          # Name of Nginx status module OHI is to query against. discover | ngx_http_stub_status_module | ngx_http_status_module | ngx_http_api_module
-          status_module: discover
+    # New users should leave this property as `true`, to identify the
+    # monitored entities as `remote`. Setting this property to `false` (the
+    # default value) is deprecated and will be removed soon, disallowing
+    # entities that are identified as `local`.
+    # Please check the documentation to get more information about local
+    # versus remote entities:
+    # https://github.com/newrelic/infra-integrations-sdk/blob/master/docs/entity-definition.md
+    REMOTE_MONITORING: true
 
-          # New users should leave this property as `true`, to identify the
-          # monitored entities as `remote`. Setting this property to `false` (the
-          # default value) is deprecated and will be removed soon, disallowing
-          # entities that are identified as `local`.
-          # Please check the documentation to get more information about local
-          # versus remote entities:
-          # https://github.com/newrelic/infra-integrations-sdk/blob/master/docs/entity-definition.md
-          remote_monitoring: true
-          # validate_certs: true 
-          # default: true If the status URL is HTTPS with a self-signed certificate, set this to false if you want to avoid certificate validation
-      labels:
-          env: production
-          role: load_balancer
+    # VALIDATE_CERTS: true 
+    # DEFAULT: true If the status URL is HTTPS with a self-signed certificate, set this to false if you want to avoid certificate validation
```

Comment:
Small detail here. 
DEFAULT is not an argument/env var. it's only a comment related to VALIDATE CERTS. 
I would change it to something like `# default true. If the. statu........` and move it up one line to to sit before the VALIDATE CERTS line to standardize as in the other comments.

All the rest looks good 👍

Reply:
Ops, I got confused by the wording of the config, fixed!